src/main/webapp/app/entities/programming-exercise/programming-exercise.component.html

```diff
@@ -14,6 +14,17 @@ <h4>
                 Link new Programming Exercise
             </span>
         </button>
+        <button
+            *ngIf="course && course.isAtLeastInstructor"
+            id="jh-import-entity"
+            class="btn btn-primary float-right jh-create-entity create-programming-exercise mr-1"
+            [routerLink]="['/', { outlets: { popup: ['course', course.id, 'programming-exercise-import'] } }]"
```

Comment:
invoke a method in the component instead of using a router link

Reply:
Using a method, which just opens a modal now